app/gimlet-rot/app.toml

```diff
@@ -118,3 +118,13 @@ pins = [
 { pin =  { port = 0, pin = 7 }, alt = 3 },
 ]
 spi_num = 5
+
+[tasks.sp_measure]
+name = "task-sp-measure"
+priority = 5
+max-sizes = {flash = 32768, ram = 8192}
+task-slots = ["swd"]
+stacksize = 2048
```

Comment:
Do we want `start = false` here?

Reply:
I don't actually know what the default is 😆

So yeah, having it explicit for documentation purposes would be nice!

Reply:
I added a comment stating that we're not starting the task